app/src/androidTest/java/ViewTestUtils.kt

```diff
@@ -46,6 +44,43 @@ fun clickBottomToolbarButton(item: Int) {
         .perform(click())
 }
 
+fun checkHintOnEditInRecyclerView(
```

Comment:
I think it shouldn't be in Utils, because it isn't util and is too complex. It is specific case and in my opinion it won't be reused. In utils hold small, reusable, atomic operations.

Reply:
This is interesting place where in bottom bar where I want scroll to next element I need use it.  I will use it in another tests.